platforms/android/src/org/crosswalk/engine/XWalkCordovaUiClient.java

```diff
@@ -172,12 +175,15 @@ public void onPageLoadStopped(XWalkView view, String url, LoadStatus status) {
     // File Chooser
     @Override
     public void openFileChooser(XWalkView view, final ValueCallback<Uri> uploadFile, String acceptType, String capture) {
-        uploadFile.onReceiveValue(null);
+        if (mFileChooser == null) {
+            mFileChooser = new XWalkFileChooser(parentEngine.cordova.getActivity());
```

Comment:
There is a table of using Context in different scenario https://possiblemobile.com/2013/06/context/,  is it mean that Application Context doesn't suite XWalkView?